samples/snippets/cloud-client/src/test/java/privateca/SnippetsIT.java

```diff
@@ -243,12 +298,14 @@ public void testEnableDisableCertificateAuthority()
   }
 
   @Test
-  public void testDeleteCertificateAuthority()
-      throws InterruptedException, ExecutionException, IOException {
-    privateca.DeleteCertificateAuthority.deleteCertificateAuthority(
+  public void testDeleteUndeleteCertificateAuthority()
```

Comment:
We need one IT per sample instead of putting all the ITs in one file. So something like `samples/snippets/cloud-client/src/main/java/privateca/ActivateSubordinateCaIT.java` this helps with debugging if a samples IT fails -- we would know right away which sample is failing. Some [examples](https://github.com/googleapis/java-bigquery/tree/master/samples/snippets/src/test/java/com/example/bigquery).

Reply:
Thanks for reviewing!
I have written test cases to support parallel execution and it would point to the exact test that's failing as they are not interlinked.